test/chunking-form/samples/basic-chunking/_expected/system/main2.js

```diff
@@ -7,21 +7,21 @@ System.register(['./chunk-b663d499.js'], function (exports, module) {
     }],
     execute: function () {
```

Comment:
With the System module format it may make sense to better flatten the surrounding wrapper so that this doesn't look quite so odd.... but note that removing these three levels of indentation is one of the goals of this PR.